Plugins/OpenAPIGenerator/plugin.swift

```
@@ -47,21 +50,14 @@ struct SwiftOpenAPIGeneratorPlugin {
         targetName: String
     ) throws -> [Command] {
         let inputFiles = sourceFiles
-        guard let config = inputFiles.first(where: { $0.path.lastComponent == "openapi-generator-config.yaml" })?.path
-        else {
+        guard let config = inputFiles.first(where: {
+            supportedConfigFiles.contains($0.path.lastComponent)
+        })?.path else {
             throw Error.noConfigFound(targetName: targetName)
         }
-        guard
-            let doc = inputFiles.first(where: {
-                switch $0.path.lastComponent {
-                case "openapi.yaml", "openapi.json":
-                    return true
-                default:
-                    return false
-                }
-            })?
-            .path
-        else {
+        guard let doc = inputFiles.first(where: {
+            supportedDocFiles.contains($0.path.lastComponent)
+        })?.path else {
```

Should we consider what happens when there are multiple files here with different extensions. 

Should we throw an error?

Similar question about the config file. Be nice to have some clear semantics, even if it’s explicitly disallowed. 

//cc @czechboy0

---

Great point, we should fail if more than one are found – but probably worth breaking out into a separate issue?

---

Done with [`7cb538a` (#77)](https://github.com/apple/swift-openapi-generator/pull/77/commits/7cb538a3079c4390eef4eb562e037d1c9776bdbc)

---

Looks like this introduce some problems in IntegrationTest. I'm currently working to fix it.